test/unit/frontend/components/form-row.spec.js

```diff
@@ -0,0 +1,197 @@
+import { mount } from '@vue/test-utils'
+import FormRow from '@/components/FormRow.vue'
+import { expect, vi } from 'vitest'
+
+describe('FormRow', () => {
+    test('mounts with type="checkbox"', async () => {
+        expect(FormRow).toBeTruthy()
+
+        const wrapper = mount(FormRow, {
+            props: {
+                type: 'checkbox'
+            },
+            shallow: true
+        })
+
+        // console.log(wrapper.html())
```

Comment:
✂️

Reply:
Assuming you're just referring to the `console.log` here, not the full 4 lines included in the comment?

Reply:
https://github.com/flowforge/flowforge/pull/1201/commits/835855731d4f8661b98627310b4f9884cc4e7289